### PR TITLE
Basic GET/POST endpoints

### PR DESCRIPTION
**Pull request by tazwar22**
**Additions:**
- Added  `gorilla/mux` as the primary routing module
- Implemented simple GET and POST request endpoints for users and articles
- Tested endpoints via [Postman](https://www.postman.com/)
- Communicated with @achinth-b in order to decide on other possible endpoints to expose (eg. PUT/DEL)
- Added and tested `prose`, an NLP library which will come in handy later on.(excluded it from `main.go` for now)

**Usage:**
- Same as before... 

Direct response to **Issue #15** 

Useful resources:
- [RESTful API development](https://tutorialedge.net/golang/creating-restful-api-with-golang/)
- [NLP Library](https://github.com/jdkato/prose)

### CODE REVIEW

File: backend/main.go [hunk 1]
```diff
@@ -4,34 +4,89 @@ import (
...
+
+    log.Fatal(http.ListenAndServe(":8090", router))
+}
+
+
+
+
 func main() {
-	if err := godotenv.Load(); err != nil {
+	if err := godotenv.Load("example.env"); err != nil {
```
Comment (tarikeshaq) [inline backend/main.go:84]: You might wanna duplicate and rename your example.env to `.env` to get it to work - we shouldn't need to load the `example.env`

File: backend/main.go [hunk 2]
```diff
@@ -4,34 +4,89 @@ import (
...
 	db.Init()
 
-	http.HandleFunc("/hello", hello)
-	http.HandleFunc("/headers", headers)
-
-	fmt.Println("Running server on port 8090")
-	http.ListenAndServe(":8090", nil)
+	fmt.Println("Starting Server...")
+	//Controls all endpoints
+	handleRequests()
```
Comment (tarikeshaq) [inline backend/main.go:91]: Awesome!!!

File: backend/main.go [hunk 3]
```diff
@@ -4,34 +4,89 @@ import (
...
+
+}
+
+
+
+
+//Driver for all routes
+func handleRequests(){
+
+	router := mux.NewRouter().StrictSlash(true)
```
Comment (tarikeshaq) [inline backend/main.go:67]: I love that we're using `mux`! It's amazing!

**Review by tarikeshaq — APPROVED** [review]
This looks great!! Good job @tazwar22

File: backend/main.go (hunk 1 above)
Comment (tazwar22) [reply backend/main.go:84]: Alright. Noted!

File: backend/main.go (hunk 3 above)
Comment (tazwar22) [reply backend/main.go:67]: It's really clean!

**Pull request merged**